### PR TITLE
EPMRPP-38852 - React: Navigation via Next/Previous buttons from log v…

### PR DESCRIPTION
**Pull request by mykhailo-kalashnikov-epam**
…iew does not work correctly for drill down via list view

### CODE REVIEW

File: app/src/controllers/testItem/sagas.js [hunk 1]
```diff
@@ -118,10 +125,10 @@ function* fetchTestItems({ payload = {} }) {
   );
   const dataPayload = yield take(createFetchPredicate(NAMESPACE));
   let level;
-  if (!parentItem) {
```
Comment (DiscoElevator) [inline app/src/controllers/testItem/sagas.js]: Why did you revert this change?

Comment (mykhailo-kalashnikov-epam) [reply app/src/controllers/testItem/sagas.js]: This change is no longer necessary, SUITE_LEVEL defined as default, also if we go to log page from statistic - level changing to LEVEL_NOT_FOUND which broke navigation logic on logPage.
